services/common/src/components/forms/RenderSelect.tsx

```diff
@@ -56,7 +57,7 @@ export const RenderSelect: FC<SelectProps> = ({
                 (meta.warning && <span>{meta.warning}</span>))
             }
             id={id}
-            getValueProps={() => ({ value: input.value })}
+            getValueProps={() => input.value !== "" && { value: input.value }}
```

Comment:
placeholder wasn't showing because I think redux was supplying it with "" when it was empty.